dftimewolf/lib/state.py

```diff
@@ -234,31 +218,24 @@ def StoreContainer(self,
     """Thread-safe method to store data in the state's store.
 
     Args:
-      container (AttributeContainer): data to store.
+      container: data to store.
       source_module: the originating module.
     """
     with self._state_lock:
       container.SetMetadata(
           interface.METADATA_KEY_SOURCE_MODULE, source_module)
       self.store.setdefault(container.CONTAINER_TYPE, []).append(container)
 
-  def StoreStats(self, stats_entry: StatsEntry) -> None:
-    """Thread-safe method to store stats in the state's stats store.
+  def LogTelemetry(self, telemetry_entry: telemetry.TelemetryEntry) -> None:
+    """Thread-safe method to store telemetry in the state's telemetry store.
```

Comment:
It probably _is_ thread safe, but with the mutex removed, probably don't need to call that out.
```suggestion
    """Method to store telemetry in the state's telemetry store.
```